app/controllers/reviews_controller.rb

```diff
@@ -3,12 +3,15 @@ class ReviewsController < ApplicationController
 
   def create
     @video = Video.find(params[:video_id])
-    review = @video.reviews.build(review_params.merge!(user: current_user))
-    if review.save
-      redirect_to @video
+    review = @video.reviews.create(review_params.merge!(user: current_user))
+    if review.errors.any?
+      review.errors.messages.values.each do |msg|
```

Comment:
I found a nagging issue though ... when using review.errors.full_messages.each do |error ... The error display in the popup with the actual model name embedded in the message string! When using the "message" option on the model this happens, this was the only way I could figure out to not display it. Any ideas how I could better accomplish this?